_clanky/volební-mapa-čr-volby-2021.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+detail: true
+title: Volební mapa ČR - volby 2021
+ref: volební-mapa-čr-volby-2021
+lang: cs
+image: ../attachments/články/volební-mapa-čr-volby-2021/obrázky/orp.svg
+author: michal_škop
+date: 2021-11-28 07:00:00 +01:00
+---
+Aš a Kraslice jsou na západním konci ČR, Karviná s Orlovou na východním. Přesto volebním chováním jsou na společném extrému v rámci ČR. Detailní klasické geografické mapy s volebními výsledky jsou již tradiční součástí volebního servisu médií. Ukážeme si proto podrobnější analýzu výsledků parlamentních voleb 2021 za pomoci otevřených dat - cílem je mapa ČR, ale ne klasická geografická, nýbrž “volební”.
+
+<!--more-->
+Klasická volební mapa s výsledky voleb po jednotlivých okrscích tentokrát vznikla v několika médiích: Mj. ji zveřejnil [Deník][link_denik], [iRozhlas][link_irozhlas] nebo [Seznam Zprávy][link_seznamzpravy]. Postup, jak na takovéto mapy, lze najít ve starším článku [Mapa s výsledky voleb po okrscích][link_mapa].
+
+{% include image.html url="../attachments/články/volební-mapa-čr-volby-2021/obrázky/mapa_sz.webp" description="Mapa s výsledky voleb po okrscích. Zdroj: <a href='https://www.seznamzpravy.cz/clanek/nejpodrobnejsi-vysledky-voleb-unikatni-mapa-vsech-okrsku-ceska-177583'>Seznam Zprávy</a>" %}
+
+### Výsledek: Volební mapa ČR dle toho, jak lidé v jednotlivých regionech volí
+
```

Comment:
@michalskop tady chybí nějaký text mezi 2 nadpisy - např. co najdeme v této sekci